src/server/rdb_load.cc

```diff
@@ -2364,9 +2364,9 @@ void RdbLoader::LoadItemsBuffer(DbIndex db_ind, const ItemsBuf& ib) {
       continue;
 
     try {
-      auto [it, added] = db_slice.AddOrUpdate(db_cntx, item->key, std::move(pv), item->expire_ms);
-      it->first.SetSticky(item->is_sticky);
-      if (!added) {
+      auto res = db_slice.AddOrUpdate(db_cntx, item->key, std::move(pv), item->expire_ms);
```

Comment:
I believe rdb load should not call rep and post update

Reply:
But is there any harm in that?

Reply:
in post update you add the memory and add some statistics. The statistic will show wrong values if we call it here.
when server loads from rdb where do we account today the memory?

Reply:
After looking again:
Before this change, `AddOrUpdate()` called `PostUpdate()` internally. With this change, it is externalized to be in the scope here. I don't think there's harm in that, right?